xmodule/html_block.py

```diff
@@ -498,3 +498,4 @@ def safe_parse_date(date):
     _ExtractedHtmlBlock if settings.USE_EXTRACTED_HTML_BLOCK
     else _BuiltInHtmlBlock
 )
+HtmlBlock.__name__ = "HtmlBlock"
```

Comment:
We need to ensure that the HTML subclasses (CourseInfoBlock, AboutBlock, and StaticTabBlock) are based on the extracted HTML block code rather than the built-in code, as they are now. This may require exposing a new HTMLMixin class from xblocks-contrib.

This can be done in a follow-up.